src/editor/codemirror/language-server/signatureHelp.ts

```diff
@@ -44,167 +45,6 @@ interface SignatureHelpState {
   result: SignatureHelp | null;
 }
 
-const signatureHelpTooltipField = StateField.define<SignatureHelpState>({
```

Comment:
This looks like a lot has changed but I've just moved it into the scope of the function below so it can access `intl`.